ROADMAP.md

```diff
@@ -1,3 +1,31 @@
+# Katib 2021 Roadmap
+
+## New Features
+
+### AutoML
+
+- Support Population Based Training [#1382](https://github.com/kubeflow/katib/issues/1382)
+- Support [ASHA](https://arxiv.org/pdf/1810.05934.pdf)
+- Support Auto Model Compression [#460](https://github.com/kubeflow/katib/issues/460)
+- Support Auto Feature Engineering [#475](https://github.com/kubeflow/katib/issues/475)
+- Various CRDs for HP, NAS and other AutoML techniques.
+
+### UI
+
+- Migrate to the new Katib UI [Project 1](https://github.com/kubeflow/katib/projects/1)
+- Hyperparameters visualization with fANOVA algorithm
```

Comment:
[nits] I'd say "Hyperparameter importances" is more precise. What do you think?

```suggestion
- Hyperparameter importances visualization with fANOVA algorithm
```

Reply:
Sure, thank you!